src/scripts/annotate.pl

```diff
@@ -807,14 +836,62 @@ sub is_bad_fusion{
 close(hFo);
 rmtree(["$annotation_dir"]);
 
+sub exist_multiplename_pair_checking {
+	my %fusionpartner_list = %{(shift)};
+	my $targetgene1 = shift;#e.g. targetgene UBTF,MIR6782
+	my $targetgene2 = shift;#e.g. targetgene UBTF,MIR6782
+	my @genes1 = split(/,|\|/, $targetgene1);
+	my @genes2 = split(/,|\|/, $targetgene2);
+
+	foreach my $g1 (@genes1) {
+		foreach my $g2 (@genes2){
+			return 1 if(exists($known_fusion_partners{$g1}{$g2}));
+		}
+	}
+
+	return 0;
+}
+
+sub is_knownfusiongenepair{
+        my @genes1_tmp = %{(shift)};
+        my @genes2_tmp = %{(shift)};
+
+        foreach my $g1 (@genes1_tmp) {
+                foreach my $g2 (@genes2_tmp) {
+                        return 1 if (exists($known_fusion_partners{$g1}{$g2}));
+                }
+        }
+
+        return 0;
+}
```

Comment:
```suggestion
```
It appears this function is no longer used. It also is nearly a duplicate of `exist_multiplename_pair_checking`.